astropy/table/tests/test_index.py

```diff
@@ -514,3 +515,24 @@ def test_invalid_updates(self, main_col, table_types, engine):
             t.loc[[1, 4, 2]] = [[1, 2, 3], [4, 5, 6], [2, 3]]
         with pytest.raises(ValueError):
             t.loc[[1, 4, 2]] = [[1, 2, 3], [4, 5], [2, 3]]
+
+
+def test_get_index():
+    t = Table()
+    # Initializing Table with two columns
+    t['a'] = [11, 2, 1, 3, 34, 23, 42, 3, 423, 4, 54, 4, 443]
+    t['b'] = [1, 2, 11, 23, 43, 3, 41, 1, 23, 4, 54, 4, 4]
+    x = None
+    # Getting the values of index
+    for index in t['a'].info.indices:
+        if set([col.info.name for col in index.columns]) == ['a']:
```

Comment:
This is failing because you are comparing `set(['a'])` to `['a']` and that is `False`.  The right set should be `set(['a'])`.